src/AppBundle/Model/User/PendingActivation.php

```diff
@@ -19,6 +19,7 @@
  * Model that represents the internal activation life cycle during the user approval.
  *
  * @author Maximilian Bosch <maximilian.bosch.27@gmail.com>
+ *
```

Comment:
such CS fixes are unnecessary and shouldn't be in thi sstory (a file less to review :D)